src/bot/bot.py

```diff
@@ -1,34 +1,23 @@
 import logging
 
-from telegram import Update
-from telegram.ext import AIORateLimiter, Application, TypeHandler
+from telegram.ext import AIORateLimiter, Application
 
-from src.bot.handlers import categories, feedback_form, menu, registration, tasks
-from src.core.logging.utils import logging_updates
-from src.settings import settings
 
-
-def create_bot() -> Application:
-    bot = Application.builder().token(settings.BOT_TOKEN).rate_limiter(AIORateLimiter()).build()
-
-    registration.registration_handlers(bot)
-    categories.registration_handlers(bot)
-    tasks.registration_handlers(bot)
-    menu.registration_handlers(bot)
-    feedback_form.registration_handlers(bot)
-    bot.add_handler(TypeHandler(Update, logging_updates))
+def create_bot(bot_token) -> Application:
+    bot = Application.builder().token(bot_token).rate_limiter(AIORateLimiter()).build()
     return bot
 
 
-async def start_bot() -> Application:
+async def start_bot(
+    bot: Application, BOT_WEBHOOK_MODE: bool, telegram_webhook_url: str, SECRET_KEY: str

```

Comment:
Капсом пишут константы, а это аргумент функции, он не может быть константой 
```suggestion
    bot: Application, weebhook_mode: bool, telegram_webhook_url: str, secret_key: str

```